converter/internal/staticconvert/testdata/integrations.river

```diff
@@ -3,10 +3,9 @@ prometheus.exporter.apache "integrations_apache_http" {
 }
 
 prometheus.scrape "integrations_apache_http" {
-	targets      = prometheus.exporter.apache.integrations_apache_http.targets
-	forward_to   = [prometheus.remote_write.integrations.receiver]
-	job_name     = "integrations/apache_http"
-	metrics_path = "integrations/apache_http/metrics"
```

Comment:
I think this is more correct without the metrics path set for Flow since it is using the targets of the exporter components.